en/installation-guide/database-initialization.rst

```diff
@@ -67,6 +67,10 @@ Restore the latest data model that also includes demo data:
   .. figure:: images/qgep_restore_messages.jpg
   
   * Check whether in Message window last line is Exit Code 0
+  If the Exit Code is 1 and the Detail reads sth like
```

Comment:
```suggestion
.. note::

  If the Exit Code is 1 and the Detail reads sth like
```

Reply:
@TriformDaniel above I wrote a suggestion to change this to a "note". You can accept it if you like.